LIFECYCLE.md

```diff
@@ -0,0 +1,16 @@
+# Lifecycle Management
+
+## Device Lifecycle scope
+
+High level requirements on a DPU/IPU or subsequently a "device"
+
+* a way to reboot a device
+  * if connected to a host, the host should be able to survive a device reset
+* a way to update a device
+  * firmware update
+  * OS update
+  * software/application update
+* a way to recover a device
+  * reset to a known good state, e.g., factory reset
```

Comment:
Recovery from over the network, or from host, or both?

Reply:
both (could be parameter in API call)

Reply:
+1, good starting point